### DISCUSSION TITLE
Is there a way to view night skies prior to 1 CE? Before the Common Era?

### DISCUSSION THREAD

**Discussion by tym0027** [Stellarium/stellarium#2618] [Q&A]
I have only played around with the web app. At least with the web app, I can't go back before "0" CE. And second, you have to press "year back" 2022 times to get to 0 CE. Are there tools in the github repo that let me query more specific spatiotemporal moments?

**Comment by gzotti** [comment]
Stellarium-web is best used with today's skies. In the Stellarium desktop version we put in years of improvements for historical application.
